venues/learningtheory.org/COLT/2019/python/post-submission-stage.py

```diff
@@ -43,50 +43,61 @@ def post_blind_note(client, original_note, conference):
         forum=None,
         signatures= [conference.id],
         writers= [conference.id],
-        readers= ['everyone'],
+        readers= [conference.id],
         content= {
             "authors": ['Anonymous'],
-            "authorids": [],
+            "authorids": [conference.id],
             "_bibtex": None
         })
 
-    paper_group_id = conference.id + "/Paper{}".format(original_note.number)
-    author_group_id = conference.id + "/Paper{}/Authors".format(original_note.number)
-
-    blind_note.content = {
-        "authors": ['Anonymous'],
-        "authorids": [author_group_id],
-    }
-
     posted_blind_note = client.post_note(blind_note)
+    conference_id = conference.get_id()
+    
+    pc_group_id = "{conference_id}/Program_Committee".format(conference_id = conference_id)
+    paper_group_id = "{conference_id}/Paper{number}".format(conference_id = conference_id, number = posted_blind_note.number)
+    author_group_id = conference.id + "/Paper{}/Authors".format(posted_blind_note.number)
+    reviewer_group_id = "{conference_id}/Paper{number}/Reviewers".format(conference_id = conference_id, number = posted_blind_note.number)
+    reviewer_group_invited_id = "{conference_id}/Paper{number}/Reviewers/Invited".format(conference_id = conference_id, number = posted_blind_note.number)
+    reviewer_group_declined_id = "{conference_id}/Paper{number}/Reviewers/Declined".format(conference_id = conference_id, number = posted_blind_note.number)
 
-    pc_group_id = "{conference_id}/Paper{number}/Program_Committee".format(conference_id = conference.get_id(), number = posted_blind_note.number)
```

Comment:
this was the original value, I guess @mohituniyal changed it to give reader permissions to all the PCs.